subgraphs/level-finance/src/modules/transaction.ts

```diff
@@ -31,11 +38,23 @@ export function transaction(
   const poolInputTokens = pool.getInputTokens();
   const idx = pool.getInputTokens().indexOf(token.id);
   const inputTokenBalances = pool.pool.inputTokenBalances;
-  if (transactionType === TransactionType.DEPOSIT) {
-    inputTokenBalances[idx] = inputTokenBalances[idx].plus(amount);
-  }
-  if (transactionType === TransactionType.WITHDRAW) {
-    inputTokenBalances[idx] = inputTokenBalances[idx].minus(amount);
+  log.warning("[Network] {}", [dataSource.network()]);
+  if (dataSource.network() === "arbitrum-one") {
```

Comment:
use a constant for "arbitrum-one"

see: https://github.com/messari/subgraphs/blob/04fc7cd2107f1456d61ca2e7d4783c2b72ee99d4/subgraphs/compound-v3/protocols/compound-v3/src/constants.ts#L138